commands/commands.js

```diff
@@ -35,37 +32,62 @@ export const askName = () => {
     return name;
 }
 
+const userDetailsArray = [];
 const storeData = () => {
-    const userDetails = JSON.stringify(finalData);
-    const readerFile = fs.existsSync(`${__dirname}/../files/${finalData.name}.json`, "utf-8");
+    userDetailsArray.length = 0
+    const checker = fs.existsSync(`${__dirname}/../files/${finalData.name}.json`)
+    if (checker) {
+        const content = fs.readFileSync(`${__dirname}/../files/${finalData.name}.json`, "utf-8")
+        const parsedContent = JSON.parse(content);
+        parsedContent.forEach(element => {
```

Comment:
See array destructuring, https://www.freecodecamp.org/news/array-destructuring-in-es6-30e398f21d10/
you could have used:
userDetailsArray = [...parsedContent]